### PR TITLE
[Druid] Restore anti-filler reporting

### PR DESCRIPTION
**Pull request by abbottmg**
Remove null default for antiFillerSpam prop which was being set *after*
the superclass's constructor was called, probably in part because this class
is not Typescript.

As indicated in previous discussion, it's probably best to stop jumping
through hoops to get this additional data into SpellbookAbility child objects
and just provide it as a parallel type created within guardiandruid, but
that is a bigger change and I am still learning the architecture of this
project, so I thought it worthwhile to stabilize this feature in the
meantime.

### CODE REVIEW

File: analysis/druidguardian/src/CHANGELOG.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { change, date } from 'common/changelog';
 import { Kettlepaw, Zeboot, g3neral, Tiboonn, Buudha } from 'CONTRIBUTORS';
 
 export default [
+  change(date(2020, 4, 7), 'Updated class syntax to re-enable AntiFillerSpam info', Kettlepaw),
```
Comment (DylanDirlam) [inline analysis/druidguardian/src/CHANGELOG.tsx]: We have a time traveler! :wink:

**Review by rp4rk — APPROVED** [review]
LGTM :)

File: analysis/druidguardian/src/CHANGELOG.tsx (hunk 1 above)
Comment (abbottmg) [reply analysis/druidguardian/src/CHANGELOG.tsx]: Ha I decided to wait for my other 4/7 change to get merged and then forgot to actually cut this PR for a bit. I don't mind postdating it!

Comment (DylanDirlam) [reply analysis/druidguardian/src/CHANGELOG.tsx]: I was mostly joking about the 2020, that's all 😛 I've done it before

Comment (abbottmg) [reply analysis/druidguardian/src/CHANGELOG.tsx]: Oh no! I didn't even notice that

**Pull request merged**